src/engine/profiler.cc

```diff
@@ -113,15 +111,12 @@ OprExecStat *Profiler::AddOprStat(int dev_type, uint32_t dev_id) {
       idx = cpu_num_ + gpu_num_;
       break;
     default:
-      LOG(FATAL) << "Unkown dev_type";
+      LOG(FATAL) << "Unknown dev_type: " << dev_type;
       return NULL;
```

Comment:
I know there's a log(fatal) here but Coverity had pointed out that memory is not freed here. Should we delete opr_stat in the default case?

Reply:
Also added deleting in DumpProfile()